website-next/components/email-signup.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+export const BasicEmailSignup = React.memo(() => {
+  return (
+    <div
+      dangerouslySetInnerHTML={{
```

Comment:
Any particular reason why this has been used?

Reply:
It's the simplest way to use raw HTML

Reply:
Can the HTML not just be pasted in as is?